volumes/volumes.go

```diff
@@ -60,6 +60,52 @@ func (v *Volumes) Create(t *task.Task) error {
 	return err
 }
 
+func (v *Volumes) GetLatest(service, project, branch string) (*task.Task, error) {
```

Comment:
Should be attached to `Storage` instead of `Volumes`. Even if they shared the same folder/structure, `Storage` is used to store Task metadata and `Volumes` is used to access volumes **mounted** inside the running service

Reply:
See https://github.com/factorysh/microdensity/blob/main/storage/storage.go#L202